components/developer/rust/Makefile

```diff
@@ -139,12 +139,33 @@ COMPONENT_INSTALL_ENV += DESTDIR=$(PROTO_DIR)
 COMPONENT_PRE_INSTALL_ACTION = \
 	mkdir -p $(PROTO_DIR)
 
+RUSTC_BIN_POST = cargo
+RUSTC_BIN_POST+= rls
+RUSTC_BIN_POST+= rustc
+RUSTC_BIN_POST+= rustdoc
+
 COMPONENT_POST_INSTALL_ACTION= \
-	( $(RM) $(PROTOUSRLIBDIR)/*.so; \
-	  $(MKDIR) $(PROTOUSRLIBDIR.$(BITS)); \
-	  $(RM) $(PROTOUSRLIBDIR.$(BITS))/*.so; \
-	  cd $(PROTOUSRLIBDIR.$(BITS)); \
-	  $(LN) -s ../rustlib/$(RUST_ARCH)/lib/*.so .; )
```

Comment:
Please add a comment about the History/Reason of the Post Install Changes. This helps other evaluate if they are still needed or not, in later versions of the rust Package

Reply:
Fixed.